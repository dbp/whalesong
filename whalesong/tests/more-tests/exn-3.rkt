#lang whalesong

(with-handlers ([(lambda (exn) #f)
                   (lambda (exn) +inf.0)])
    (car 17))
