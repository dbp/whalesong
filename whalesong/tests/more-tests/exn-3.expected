Error: car: expected pair as argument 1 but received 17
